dashboard/modules/stats_collector/stats_collector_head.py

```diff
@@ -87,7 +97,60 @@ async def get_node(self, req) -> aiohttp.web.Response:
         node_id = req.match_info.get("node_id")
         node_info = await DataOrganizer.get_node_info(node_id)
         return await dashboard_utils.rest_response(
-            success=True, message="Node detail fetched.", detail=node_info)
+            success=True, message="Node details fetched.", detail=node_info)
+
+    @routes.get("/memory/memory_table")
+    async def get_memory_table(self, req) -> aiohttp.web.Response:
+        group_by = req.query.get("group_by")
+        sort_by = req.query.get("sort_by")
+        kwargs = {}
+        try:
+            if group_by:
+                kwargs["group_by"] = GroupByType(group_by)
+            if sort_by:
+                kwargs["sort_by"] = SortingType(sort_by)
+        except ValueError as e:
+            return await dashboard_utils.rest_response(
+                success=False, message=str(e))
```

Comment:
If we remove the `try ... except ...`, then the response will be `rest_response(success=False, message=<traceback with stack>)`.